spec/features/slug_generation_spec.rb

```diff
@@ -1,63 +1,51 @@
 require 'rails_helper'
-require 'capybara/rails'
-require 'gds_api/publishing_api_v2'
 
-RSpec.describe "creating guides", type: :feature do
-  let(:api_double) { double(:publishing_api) }
-
-  before do
-    create(:guide_community)
-  end
-
-  let :slug_examples do
-    {
-      "two words": "two-words",
-      "slug--with-----hyphens": "slug-with-hyphens",
-      "       space    slugs  ": "space-slugs",
-      'other things !@#$%^&*()_-+=/\\': "other-things",
-    }
-  end
-
-
-  describe "slug generation" do
-    context "guide that has not been published" do
-      let :guide do
-        create(:guide, :with_draft_edition, :with_topic_section)
+RSpec.describe 'Generating slugs', type: :feature, js: true do
```

Comment:
It looks like all the examples had js enabled individually but safer and better to enable it for the block :+1: